olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml

```diff
@@ -337,6 +337,13 @@ spec:
                 - consoleclidownloads
               verbs:
                 - "*"
+            - apiGroups:
+                - config.openshift.io
+              resources:
+                - clusteroperators
+                - clusteroperators/status
+              verbs:
+                - "*"
```

Comment:
We need only GET/WATCH, right? We shouldn't declare we change those.

Reply:
We can make it more stricter I guess the only reason I didnt do it earlier is because we need to review all those permissions  in the csv (we have many others with `*`) but let me check if get, watch is enough.

Reply:
Let's do the capabilities in a follow up no rush for 1.26.

Reply:
We can use `*` for our types like `KnativeServing`, or other things that we actually create like `ConsoleCLIDownload`. Here, we just read, so we should limit the permissions.

But, this isn't a pressing issue. We can fix those permissions in a follow-up PR.

Reply:
> review all those permissions

Fine w/ doing this at a later stage...

Reply:
You need also "list' let me check first and btw unfortunately we have other permissions open for non Knative stuff like service monitors etc;)